tmail-backend/jmap/extensions-cassandra/src/main/scala/com/linagora/tmail/james/jmap/settings/CassandraJmapSettingsDAO.scala

```diff
@@ -0,0 +1,124 @@
+package com.linagora.tmail.james.jmap.settings
+
+import com.datastax.oss.driver.api.core.`type`.DataTypes.{TEXT, UUID, frozenMapOf, mapOf}
+import com.datastax.oss.driver.api.core.`type`.codec.registry.CodecRegistry
+import com.datastax.oss.driver.api.core.`type`.codec.{TypeCodec, TypeCodecs}
+import com.datastax.oss.driver.api.core.cql.{BoundStatementBuilder, PreparedStatement, Row}
+import com.datastax.oss.driver.api.core.{CqlIdentifier, CqlSession}
+import com.datastax.oss.driver.api.querybuilder.QueryBuilder.{bindMarker, deleteFrom, insertInto, selectFrom, update}
+import com.datastax.oss.driver.api.querybuilder.relation.Relation.column
+import com.google.common.collect.ImmutableMap
+import com.linagora.tmail.james.jmap.settings.JmapSettings.{JmapSettingsKey, JmapSettingsValue}
+import javax.inject.Inject
+import org.apache.james.backends.cassandra.components.CassandraModule
+import org.apache.james.backends.cassandra.utils.CassandraAsyncExecutor
+import org.apache.james.core.Username
+import org.apache.james.jmap.core.UuidState
+import reactor.core.scala.publisher.{SFlux, SMono}
```

Comment:
Hmmm I think you might need some import configuration on your IDE, I'm not convinced this is gonna pass the checkstyle